Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -43,9 +43,13 @@ class PaymentSheetTestPlayground: UIViewController {
     @IBOutlet weak var linkSelector: UISegmentedControl!
     @IBOutlet weak var loadButton: UIButton!
     @IBOutlet weak var customCTALabelTextField: UITextField!
+    // Used for deferred workflows
+    // Normal: Normal client side confirmation non-deferred flow
+    // Def CSC: Deferred client side confirmation
+    // Def SSC: Deferred server side confirmation
+    // Def MC: Deferred server side confirmation with manual confirmation
+    // Def MP: Deferred multiprocessor flow
```

Comment:
I wish we could test this one without a UI test, since it's so niche.

Reply:
Oh, there are no tests. Should we add one?

Reply:
oof, the multiprocessor tests had a typo where it was using manual confirmation not multiprocessor. Fixed.